sunpy/map/mapbase.py

```diff
@@ -935,7 +935,7 @@ def resample(self, dimensions, method='linear'):
 
         References
         ----------
-        * `Rebinning <http://www.scipy.org/Cookbook/Rebinning>`_ (Original
+        * `Rebinning <http://scipy-cookbook.readthedocs.io//>`_ (Original
```

Comment:
I believe this should be:
`https://scipy-cookbook.readthedocs.io/items/Rebinning.html`